core/src/main/resources/org/apache/spark/ui/static/historypage.js

```diff
@@ -31,9 +31,20 @@ function makeIdNumeric(id) {
   return resl;
 }
 
+function padLeft(val) {
+  if(val < 10) return "0" + val;
+  else return val;
+}
+
 function formatDate(date) {
-  if (date <= 0) return "-";
-  else return date.split(".")[0].replace("T", " ");
+  if (date <= 0) {
+     return "-";
+  } else {
+     var localDate = new Date(date.split(".")[0].replace("T", " ") + " UTC");
```

Comment:
it doesn't support FF and ie,because FF can't recognize "-",it can only recognize "/"  when we use date form.
use "var localDate = new Date((date.split(".")[0].replace("T", " ")+ " UTC").replace(/-/g,"/")); "to make a date form, like "2016/09/06 10:00:00"